src/components/card.js

```diff
@@ -1,27 +1,22 @@
 import React from "react";
 import "./card.css";
 import { Card as BsCard } from "react-bootstrap/";
-import { SlLogin } from "react-icons/sl";
-
-function Card({ title, content, label }) {
-  const getArrowDirection = () => {
-    return content >= 150 ? "up" : "down";
-  };
 
+function Card({ title, content, label, isPositive, icon }) {
   const getArrowColor = () => {
-    return getArrowDirection() === "up" ? "#009E0F" : "#CC0000";
+    return isPositive ? "#009E0F" : "#CC0000";
```

Comment:
try to return the `arrow` and the `color` in this function and store in a variable

```
 const getArrowAndColor = () => {
    return footer?.isPositive ? {
      color: "#009E0F",
      arrow: "↑",
    } : {
      color: "#CC0000",
      arrow: "↓"
    };
  };

 const arrowAndColor = getArrowAndColor();

// from here you can use
arrowAndColor.color and arrowAndColor.arrow
```